system/Encryption/Exceptions/EncryptionException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace CodeIgniter\Encryption\Exceptions;
+
+use CodeIgniter\Exceptions\ExceptionInterface;
+
+/**
+ * Encryption exception
+ */
+class EncryptionException extends \RuntimeException implements ExceptionInterface
+{
+
+	public static function forDriverNotAvailable(string $driver = null)
+	{
+		return new static(lang('exception.driverNotAvailable', [$driver]));
```

Comment:
Doesn't `encryption` as the language file need to be ucfirst to not die on case-sensitive systems?

Reply:
oops - fixed